recipes/_nvidia_install.rb

```diff
@@ -18,10 +18,7 @@
 
   case node['platform_family']
   when 'rhel'
-    yum_package node['cfncluster']['kernel_devel_pkg']['name'] do
-      version node['cfncluster']['kernel_devel_pkg']['version']
```

Comment:
is there a 'version' set somewhere that we should remove to not confuse ourselves later?

Reply:
https://github.com/awslabs/cfncluster-cookbook/search?utf8=%E2%9C%93&q=%22yum_package%22&type=
https://github.com/awslabs/cfncluster-cookbook/search?utf8=%E2%9C%93&q=kernel_devel_pkg&type=

I don't see any other place where version is pin'ed with yum install.